pom.xml

```diff
@@ -20,105 +21,20 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>sawtooth</groupId>
-    <artifactId>sdk</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <artifactId>sawtooth-sdk-java</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>sawtooth-sdk-protos</module>
```

Comment:
[nitpick]: Question regarding naming convention
Why is following convention followed
~/.m2/repository/sawtooth/sawtooth-sdk-transaction-processor, .../sawtooth-sdk-signing, .../sawtooth-sdk-protos
instead of something like
~/.m2/repository/sawtooth-sdk-java/transaction-processor, .../signing, .../protos

Reply:
I'm not entirely sure. I do like that the jar comes out as `/target/sawtooth-sdk-signing-*.jar`